_docs/install-and-upgrade.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: documentation
+title: Install and upgrade
+order: 2
+redirect_from:
+  - /docs/getting_started/install.html
+---
+
+# Install and upgrade
+
+## Debian and Ubuntu based distributions
+
+First, make sure [Node.js](https://nodejs.org/) v6.13.0 or more recent is installed
+with `node --version`. If not, follow the instructions given on the
+[official documentation](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions).
+
+Then download `thelounge.deb` located at the bottom of
+[the latest release](https://github.com/thelounge/thelounge/releases/latest).
+
+Finally, open a terminal and install the downloaded package using:
+
+```
+sudo dpkg --install thelounge.deb
+```
+
+{: .alert.alert-info role="alert"}
+This methods require root access. Install The Lounge using [Yarn from an npm release](#from-npm-releases) if you do cannot use `sudo`.
+
+This also sets up a `systemd` service, enabled during the install, that you can
+control with `systemctl status|start|restart|stop|...`.
+
+The Lounge is now up and running **in private mode** at <http://localhost:9000>.
+
+Its configuration file is located at `/etc/thelounge/config.js`. To configure
+The Lounge, go to [the configuration section](/docs/server/configuration.html).
+
+To upgrade The Lounge, simply follow these steps again after downloading a new
+`thelounge.deb` file, and restart the service.
+
+You might want to serve The Lounge behind a reverse proxy such as Nginx. To know
+more about the benefits and steps, follow
+[the guide on reverse proxies ](/docs/guides/reverse-proxies.html).
+
+## Arch Linux
+
+The Lounge is [available in the AUR](https://aur.archlinux.org/packages/thelounge/).
+
+Please follow the
+[Arch Linux documentation](https://wiki.archlinux.org/index.php/Arch_User_Repository)
+to install this package from the AUR. For example, to install it using an AUR
+helper:
+
+```
+pacaur -aS thelounge
+yaourt -aS thelounge
```

Comment:
I believe these 2 commands are choices, you only have to run either of them (depending on what you use), not both.

Reply:
Right, I did not convey that clearly at all.

However, some light reading showed me that almost all managers on Arch are now unmaintained and not recommended, and that applies to both Pacaur and Yaourt 🙄
I switched to suggesting [Yay](https://github.com/Jguer/yay) which seems to be what people previously using Pacaur/Yaourt suggest, and the commands seem to be the same.

If Arch people could confirm (@maxpoulin64?), that'd be great, but otherwise it seems rather safe-ish to go with this. You good with [this](https://deploy-preview-104--thelounge.netlify.com/docs/install-and-upgrade#arch-linux) now, @xPaw?